enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/NotificationAcceptanceTest.scala

```diff
@@ -1,629 +0,0 @@
-
-    result.notifications should equal(Set(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person", "Jedi").asJava))))
-  }
-
-  test("should not warn when using dynamic property lookup with no labels") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute("EXPLAIN MATCH (n) WHERE n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications shouldBe empty
-  }
-
-  test("should warn when using dynamic property lookup with both a static and a dynamic property") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute(
-      "EXPLAIN MATCH (n:Person) WHERE n.name = 'Tobias' AND n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications should equal(Set(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person").asJava))))
-  }
-
-  test("should not warn when using dynamic property lookup with a label having no index") {
-    graph.createIndex("Person", "name")
-    createLabeledNode("Foo")
-
-    val result = innerExecute("EXPLAIN MATCH (n:Foo) WHERE n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications shouldBe empty
-  }
-
-  test("should not warn for eager before load csv") {
-    val result = innerExecute(
-      "EXPLAIN MATCH (n) DELETE n WITH * LOAD CSV FROM 'file:///ignore/ignore.csv' AS line MERGE () RETURN line")
-
-    result should use("LoadCSV", "Eager")
-    result.notifications.map(_.getCode) should not contain "Neo.ClientNotification.Statement.EagerOperatorWarning"
-  }
-
-  test("should warn for eager after load csv") {
-    val result = innerExecute(
-      "EXPLAIN MATCH (n) LOAD CSV FROM 'file:///ignore/ignore.csv' AS line WITH * DELETE n MERGE () RETURN line")
-
-    result should use("LoadCSV", "Eager")
-    result.notifications.map(_.getCode) should contain("Neo.ClientNotification.Statement.EagerOperatorWarning")
-  }
-
-  test("should not warn for load csv without eager") {
-    val result = innerExecute(
-      "EXPLAIN LOAD CSV FROM 'file:///ignore/ignore.csv' AS line MATCH (:A) CREATE (:B) RETURN line")
-
-    result should use("LoadCSV")
-    result.notifications.map(_.getCode) should not contain "Neo.ClientNotification.Statement.EagerOperatorWarning"
-  }
-
-  test("should not warn for eager without load csv") {
-    val result = innerExecute("EXPLAIN MATCH (a), (b) CREATE (c) RETURN *")
-
-    result should use("Eager")
-    result.notifications.map(_.getCode) should not contain "Neo.ClientNotification.Statement.EagerOperatorWarning"
-  }
-
```

Comment:
It seems this test is gone too

Reply:
Skipped it due to a very similar test already existed but I guess it doesn't hurt to have both so added now.